tgui/packages/tgui-panel/styles/chat-format/chat-dark-theme.scss

```diff
@@ -0,0 +1,62 @@
+/*****************************************
+*
+* cf means "chat format"
+* cfc means "chat format color" which means only dedicated for color
+*
+******************************************/
+
+// Standard colors
+// color names are not really accurate due to theme base color visibility. get the feeling.
+.cfc_red {
+  color: #e21919;
+}
+.cfc_redorange {
+  color: #e04a0e;
+}
+.cfc_orange {
+  color: #dd8716;
+}
+.cfc_yellow {
+  color: #e2df10;
+}
+.cfc_lime {
+  color: #86e608;
+}
+.cfc_green {
+  color: #0edb07;
+}
+.cfc_cyan {
+  color: #07dbad;
+}
+.cfc_bluesky {
+  color: #07aadb;
+}
+.cfc_darkbluesky {
+  color: #2d87a1;
+}
+.cfc_navy {
+  color: #075cdb;
+}
+.cfc_blue {
+  color: #0b07db;
+}
+.cfc_indigo {
+  color: #4707db;
+}
+.cfc_purple {
+  color: #900be9;
+}
+.cfc_violet {
+  color: #c90be2;
+}
+.cfc_magenta {
+  color: #d402a7;
+}
+.cfc_redpurple {
+  color: #db075f;
+}
+
+// specific chat styles
+.cfc_carpspeak {
```

Comment:
Why is carpspeak in here? This should just be a regular chat span along with all of the other ones used for the name colouring.
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/a6f2aa2e-2829-4058-b859-0a7afe5bea07)

Reply:
Those ones really should be split into their own file, but thats not a blocking change for this PR and is out of scope.

Reply:
I didn't like the chat classes are in /goon folder. I can immigrate those into the new files but I am not certain it's really fine.

Reply:
Hmm, I think immigrating classes into the files when these have prefix will work well. So, the old chat style classes in goon folder will be gone when it's over.

Reply:
The only blocking request here is to move carpspeak to the right place

Reply:
I still believe that should be in the new file, but okay. it's moved to the goon file.

Reply:
It's not a colour, its a chat speaker span, so it should be with the other chat speaker spans

Reply:
> didn't like the chat classes are in /goon folder. I can immigrate those into the new files but I am not certain it's really fine.

We are currently in a movement to kill that folder(Pigeon's efforts especially). If you don't absolutely need to put it there, then I would move it out